src/test/java/xyz/vegaone/easytracking/tag/TagServiceTest.java

```diff
@@ -0,0 +1,169 @@
+package xyz.vegaone.easytracking.tag;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import xyz.vegaone.easytracking.dto.Bug;
+import xyz.vegaone.easytracking.dto.Tag;
+import xyz.vegaone.easytracking.dto.Task;
+import xyz.vegaone.easytracking.dto.UserStory;
+import xyz.vegaone.easytracking.service.BugService;
+import xyz.vegaone.easytracking.service.TagService;
+import xyz.vegaone.easytracking.service.TaskService;
+import xyz.vegaone.easytracking.service.UserStoryService;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class TagServiceTest {
+
+    public static final String TAG_NAME = "Tag name";
+    public static final String TAG_NEWNAME = "Tag name";
+
+    public static final String BUG_DESCRIPTION = "Bug description";
+    public static final String BUG_OWNER = "Bug owner";
+    public static final Integer BUG_PRIORITY = 1;
+    public static final String BUG_STATUS = "Bug status";
+    public static final String BUG_TITLE = "Bug title";
+
+    public static final String TITLE = "Test title";
+    public static final String DESCRIPTION = "Test description";
+    public static final String OWNER = "Test OWNER";
+    public static final Integer ESTIMATION = 123456;
+    public static final Integer PRIORITY = 1;
+    public static final Long PROJECT_ID = 1L;
+    public static final String STATUS = "Temporary status";
+
+    private static final String TASK_DESCRIPTION = "Task description";
+    private static final String TASK_OWNER = "Task owner";
+    private static final Integer TASK_PRIORITY = 1;
+    private static final String TASK_STATUS = "Task status";
+    private static final String TASK_TITLE = "Task title";
+
+    @Autowired
+    private BugService bugService;
+
+    @Autowired
+    private UserStoryService userStoryService;
+
+    @Autowired
+    private TaskService taskService;
+
+    @Autowired
+    private TagService tagService;
+
+    @Test
+    public void createTagTest(){
+        //given
+        Tag tag = new Tag();
```

Comment:
I see that this block of code is used in all the tests. Could you extract it in a private method?